src/docs/guide/interactivity.md

```diff
@@ -7,4 +7,4 @@ order: 7
 show_guide: true
 ---
 
-Coming soon.
+Coming soon. See the [cursor component docs](../components/cursor.html) in the meantime.
```

Comment:
ok i just changed it since these will be removed soon anyways. to anti-bikeshedding